spring-cloud-generator/excluded_libs.txt

```diff
@@ -0,0 +1,5 @@
+# this is a list of libraries to be ignored by generate-library-list.sh
+google-cloud-speech
+google-cloud-dialogflow
+google-cloud-dataproc
+google-cloud-dataplex
```

Comment:
Does `apigee-connect` mentioned in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1407#issuecomment-1363280649 belong to this list?

Reply:
Will check

Reply:
It should, but this PR will likely get closed - see https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1423#issuecomment-1371508894